TuneUp/TuneUpWindowViewModel.cs

```diff
@@ -610,20 +613,16 @@ private void CreateGroupNodesForCollection(ObservableCollection<ProfiledNodeView
                     var pGroup = new ProfiledNodeViewModel(pNode)
                     {
                         GroupExecutionOrderNumber = executionCounter++,
-                        GroupExecutionMilliseconds = groupExecTime
+                        GroupExecutionMilliseconds = groupExecTime,
```

Comment:
Can you add some comments here to provide why we need nodesToAdd here?. 
I thought this was fixed by https://github.com/DynamoDS/Dynamo/pull/15547

Reply:
I've written in the description that I couldn't build from master so I created this branch from DYN-7697-Crash-with-Periodic-Run as it was the latest I've published (the branch of #15547). Relevant changes to this ticket are only in CreateGroupNodesForCollection.